package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "export NODE_OPTIONS=--openssl-legacy-provider && set export NODE_OPTIONS=--openssl-legacy-provider && next dev",
-    "build": "export NODE_OPTIONS=--openssl-legacy-provider && export NODE_OPTIONS=--openssl-legacy-provider && next build",
-    "start": "export NODE_OPTIONS=--openssl-legacy-provider && export NODE_OPTIONS=--openssl-legacy-provider && next start",
+    "dev": "export NODE_OPTIONS=--openssl-legacy-provider && next dev",
```

Comment:
Huh, wonder the package.json shows a diff here? Once my PR is merged, I can pull `main` into this PR for ya to see if this still appears.

Reply:
Ooop - thought I fixed this! I will reset it back to your version in my next commit